app/views/pages/scholarships.html.erb

```diff
@@ -45,6 +45,8 @@
         <h2>Software conference scholarships</h2>
         <p class="lead">We are proud to partner with Ruby on Ales and GitHub, respectively, to introduce more transitioning military, citizen-soldiers, veterans and their families to software development through conferences. To apply, send our Scholarship Chair, Nell Shamrell-Harrington, a direct message on Slack with 2-3 sentences on your interest. Upcoming 2016 software conferences:</p>
         <ul>
+          <li><p class="lead"><a href="https://360idev.com/">360iDev</a>, August 21-24, 2016</li></p>
+          <li><p class="lead"><a href="https://www.codeonthebeach.com/">Code on the Beach</a>, Atlantic Beach, FL, August 12-14, 2016</li></p>
           <li><p class="lead"><a href="http://codeconf.com">CodeConf</a>, Los Angeles, June 27-29, 2016</li></p>
           <li><p class="lead"><a href="http://git-merge.com">GitMerge</a>, New York City, April 5, 2016</li></p>
           <li><p class="lead"><a href="http://ruby.onales.com">Ruby on Ales</a>, March 31 & April 1, 2016</li></p>
```

Comment:
@nellshamrell Since they've already occured, I think we can remove the links to CodeConf, GitMerge, & Ruby on Ales.

Or perhaps just add some CSS styling to indicate that they've already occurred (maybe change text or background color / grey them out / italicize / or add a strikethrough?) - it might be nice to have a list for the year that includes past scholarships.